docs/i18n.md

```diff
@@ -84,3 +84,113 @@ the JSON to the repo.
 # build the JSON.
 NODE_APP_INSTANCE=[MY_APP] bin/build-locales
 ```
+
+## Setting up translations
+
+To set-up a component to be translated there are two pieces of code to know
+about.
+
+### Jed
+
+We use [Jed](https://slexaxton.github.io/Jed/) as the API for providing
+`gettext` functions inside React components. An initialized `Jed` instance
+has all the `gettext` related functionality exposed as methods. There is a
+fancy chained API but we've stuck to a more traditional approach.
+
+Before we get into how to make use of these functions let's take a look at
+how the Jed instance is exposed to our components.
+
+### The Translation Provider
+
+The translation provider is used to pass down a Jed instance via context to
+components lower down in the component hierarchy. This part is already done
+for you in the addons-frontend code-base. So you should only need to worry
```

Comment:
"code-base" seems odd to me.

"This part is already done for you in addons-frontend" would be fine.